inductiva/coastal/test.py

```diff
@@ -0,0 +1,25 @@
+"""End to end CoastalArea scenario simulation"""
+
+import inductiva
+import matplotlib.pyplot as plt
+
+bathymetry = inductiva.coastal.Bathymetry.from_random_depths(x_range=(0, 100),
+                                                             y_range=(0, 100),
+                                                             x_num=100,
+                                                             y_num=100,
+                                                             max_depth=50,
+                                                             random_seed=12)
+
+bathymetry.plot()
+plt.show()
```

Comment:
Ah! Comment on this pls!
But we have a readability/pattern problem here.
The method  plot() gives *no* hint that we are using matplotlib, or that we then need to do

```
import matplotlib.pyplot as plt

plt.show()
```

It does not seem reasonable to assume that the user will know how to do that (I wasn't able to figure it out because I was really NOT expecting this pattern).

Either we have:
```
bathymetry.show()
```
that internally calls plot and plt.show without the user having to do imports or we allow plot to have an extra argument that allows us to optiobally show
```
bathymetry.plot(show=True)
```

@hpenedones : what do you think? We may need to go over the patterns we are using for visualizing objects, and there seems to be two different situations (basic / simple visualization such as this one, and more complex postprocessing / visualization methods). The simple ones, that require only matplotlib or something very standard can exist in the client side, but we need to make sure we follow a consistent way of calling them.